toolchain-fitnesse-plugin/src/test/java/nl/praegus/fitnesse/responders/testHistory/RecentTestHistoryResponderTest.java

```diff
@@ -62,6 +63,24 @@ public void Checks_number_of_times_passed(){
 
         assertThat(recentTestHistory.getHistoryLines().get(0).getNumberOfTimesPassed()).isEqualTo(0);
     }
+    @Test
+    public void Checks_if_filter_is_off(){
+        RecentTestHistory recentTestHistory = new RecentTestHistory(getMockDir("TestResultDirectory"));
+        List<String> expectedResult = Arrays.asList("Example.Mocker3.SuiteSetUp","Example.Mocker3.SuiteTearDown","Example.Mocker3.SetUp","Example.Mocker3.TearDown");
```

Comment:
Is het niet logisch om in het expected result ook een normale pagina te hebben? je wil toch ook zien dat je die ook terugkrijgt?